music_assistant/server/providers/radiobrowser/__init__.py

```diff
@@ -162,7 +157,7 @@ async def browse(self, path: str, offset: int, limit: int) -> list[MediaItemType
         if subpath == "country":
             items: list[BrowseFolder | Radio] = []
             for country in await self.radios.countries(
-                order=Order.NAME, hide_broken=True, limit=limit, offset=offset
```

Comment:
why did you remove the paging ? Is it broken in the library ?

Reply:
Yeah paging with order Is broken. As I explained in the description it doesn't properly sort the station name. Since it gets cached it should not be a problem unless you see otherwise

Reply:
the paging does not need the order. Sorting and filtering is done by the frontend.

Reply:
bit confused, I put some logging and observed it was sending:
offset=0, limit=50
offset=50, limit=100
...

but if you say the frontend is doing the sorting, there is something not working because it wasnt doing it.  I synced my  fork with main before starting last night, so I am not sure. 
I will have to test again if you think that is true.